molior/api/build.py

```diff
@@ -24,9 +24,11 @@ async def get_builds(request):
     description: Returns a list of builds.
     tags:
         - Builds
-    consumes:
-        - application/x-www-form-urlencoded
```

Comment:
I guess the default is json?

Reply:
That's correct. If nothing is specified, Swagger UI uses json. Maybe it could be made clearer by having some global mime setting: https://swagger.io/docs/specification/2-0/mime-types/